tests/app/main/views/test_set_lang.py

```diff
@@ -0,0 +1,52 @@
+from uuid import uuid4
```

Comment:
It doesn't look to me like any tests are being run in CI in this repo. That's not an issue with this PR specifically, but would be nice to fix.

Reply:
Added CI in https://github.com/cds-snc/notification-document-download-frontend/pull/30 and I'll rebase